highfive/newpr.py

```diff
@@ -221,14 +230,20 @@ def choose_reviewer(repo, owner, diff, exclude, config):
                 end = line.find("/", start)
                 if end == -1:
                     continue
+                full_end = line.rfind("/", start)
 
                 cur_dir = line[start:end]
+                full_dir = line[start:full_end] if full_end != -1 else ""
 
                 # A few heuristics to get better reviewers
                 if cur_dir.startswith('librustc'):
                     cur_dir = 'librustc'
                 if cur_dir == 'test':
                     cur_dir = None
+                if len(full_dir) > 0:
+                    for entry in mentions:
+                        if entry.startswith(full_dir) and entry not in to_mention:
```

Comment:
Should this be equal rather than startswith? Or if it is startswith, I would expect full_dir.startswith(entry)